### DISCUSSION TITLE
How to make target temperature persistent?

### DISCUSSION THREAD

**Discussion by degreal** [ScratMan/HASmartThermostat#127] [Q&A] [answered]
Every time I update HA all smart thermostats reset to whatever is specified in "target_temp" parameter. How to make target more persistent?

**Comment by ScratMan** [comment, marked as answer]
Hello, bug fixed by commit https://github.com/ScratMan/HASmartThermostat/commit/f40e1bf0dfb12c078b6410be5d07f3a2f2930b9a
